doc/cross-compile/linux-arm.md

```diff
@@ -0,0 +1,243 @@
+# Cross-compiling iceoryx2
+
+## Cross-compile project with `cross`
+
+Setup `cross` as described here: [cross-rs GitHub](https://github.com/cross-rs/cross).
+
+Once `cross` is setup, navigate to root of the iceoryx2 repo and run
+the command below.
+
+```bash
+cross build --target aarch64-unknown-linux-gnu --release --package iceoryx2-ffi
+```
+
+## Build C bindings
+
+Run the following at the repo root:
+
+```bash
+cmake -S . -B target/ffi/build -DBUILD_EXAMPLES=OFF -DCMAKE_INSTALL_PREFIX=target/ffi/install -DRUST_TARGET_TRIPLET='aarch64-unknown-linux-gnu' -DBUILD_CXX_BINDING=OFF -DRUST_BUILD_ARTIFACT_PATH="$( pwd )/target/aarch64-unknown-linux-gnu/release"
```

Comment:
When `DRUST_BUILD_ARTIFACT_PATH` is used, the `DRUST_TARGET_TRIPLET` is superfluous
```suggestion
cmake -S . -B target/ffi/build -DBUILD_EXAMPLES=OFF -DCMAKE_INSTALL_PREFIX=target/ffi/install -DBUILD_CXX_BINDING=OFF -DRUST_BUILD_ARTIFACT_PATH="$( pwd )/target/aarch64-unknown-linux-gnu/release"
```

If you have time, it would also be great if you could add the C++ bindings. If not, it's a task for another developer.